src/Stryker.Core/Stryker.Core/Mutators/StringEmptyMutator.cs

```diff
@@ -5,112 +5,108 @@
 using Microsoft.CodeAnalysis.CSharp.Syntax;
 using Stryker.Core.Mutants;
 
-namespace Stryker.Core.Mutators
+namespace Stryker.Core.Mutators;
+
+/// <summary>
+/// Mutator that will mutate:
+/// <para>the access to <c>string.Empty</c> to a string that is not empty.</para>
```

Comment:
```suggestion
/// <param>the access to <c>string.Empty</c> to a string that is not empty.</param>
```

Reply:
pretty sure those are not param tags; no idea why they are here. I am not the author here, I think I merely moved the class to a dedicated file or someting